paper-server/patches/sources/net/minecraft/world/level/block/entity/vault/VaultBlockEntity.java.patch

```diff
@@ -85,3 +94,12 @@
          }
  
          private static List<ItemStack> resolveItemsToEject(ServerLevel level, VaultConfig config, BlockPos pos, Player player, ItemStack key) {
+             LootTable lootTable = level.getServer().reloadableRegistries().getLootTable(config.lootTable());
++        // Paper start - Add loot table to event
++            return resolveItemsToEject(level, config, pos, player, key, lootTable);
++        }
++        private static List<ItemStack> resolveItemsToEject(ServerLevel level, VaultConfig config, BlockPos pos, Player player, ItemStack key, LootTable lootTable) {
++        // Paper end
```

Comment:
Make sure your `// .. end` comments include the same "notation" as the start one so they can be linked to each other. E.g. `// Paper end - Add loot table to event`